src/components/CR8 - GalleryText/GalleryText.vue

```diff
@@ -0,0 +1,234 @@
+<template>
+    <div class="gallery-text">
+        <!-- Title -->
+        <h2 class="title">Our Ecosystem</h2>
+
+        <!-- Subtitle w/ Arrow -->
+        <div class="subtitle">
+            <span class="subtitle-text">We develop and produce original content for all genres and formats.</span>
+            <span class="prompt">
+                <span class="prompt-text" >EXPLORE</span>
```

Comment:
Never UPPERCASE in markup. Use CSS to do this. Think of markup as it should be readable and correct grammar if there was no CSS.